APMrover2/APMrover2.cpp

```diff
@@ -320,6 +320,12 @@ void Rover::update_GPS(void)
 
 void Rover::update_current_mode(void)
 {
+    // check for emergency stop
```

Comment:
sorry to be annoying but can we move this to Steering.cpp's Rover::set_servos() function?